eco.Rmd

```diff
@@ -5,8 +5,11 @@
 This chapter assumes you have a strong grasp of spatial data analysis and processing, covered in chapters 2-5.
 In it you will make use of R's interfaces to dedicated GIS software, and spatial cross validation, topics covered in chapters xx and xx respectively.
 
+## Introduction
+
 keywords: species distribution modeling, floristic modeling, spatial cross-validation, spatial autocorrelation, ordination (NMDS, Isomap), predictive mapping
```

Comment:
This seems like a good set of topics to cover. My only concern would be that in attempting to demonstrate so many things it looses depth. Slow, steady coverage of a few topics in depth would be preferable trying to squeeze too much in. That will allow you to be expansive in the prose and guide people to more advanced books, focussing on the geocomputing aspect of it, rather than trying to cover everything.

Reply:
We can shorten the title even further, Ecological use case/example is fine. 
Your suggested title "Predictive mapping of species distributions", however, doesn't capture what I would like to do, which is predictive mapping of floristic composition.
Don't worry, I will not go into detail with respect to each method. Rather I want to show an ordination, and how to model the corresponding scores. Of course, we have to use spatial cv but we don't have to show that in minute detail since I will write one chapter on it.